src/getrf_tntpiv.cc

```diff
@@ -255,7 +265,13 @@ void getrf_tntpiv(
                     A.tileBcast(
                         k, j, A.sub( k+1, A_mt-1, j, j ),
                         Layout::ColMajor, tag_j );
+                }
 
+                #pragma omp task depend(in:column[k]) \
+                                 depend(inout:column[j]) \
```

Comment:
Since this task and the previous task are both inout:column[j], it seems hard to see why there is an advantage to breaking them up?

Reply:
The previous task doesn't depend on `column[k]`.  So, it can start before the `trsm` for the left panel is completed.  Whereas this task has to wait for the column to complete (since it's the Schur-complement).